test/test_helpers/httptestserver.cpp

```diff
@@ -75,4 +79,13 @@ void HttpTestServer::removeEndpoint(const std::string& mockId)
 	process.writeLine(mockId);
 }
 
+std::uint32_t HttpTestServer::num_Hits(HttpTestServer::MockRegistration& mockRegistration)
```

Comment:
Maybe some refactoring went slightly wrong here, the rest of the methods use regular camelCase without underscores. 8-)

Reply:
Indeed, I guess I got confused and mixed up somewhere between Rust's style and the styling I'm used to at my current job.
Thanks, will fix in a moment